content/blog/vscode-esp-idf-v1.8.0/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "ESP-IDF VS Code Extension v1.8.0 release"
+date: 2024-07-12T16:07:09+08:00
+showAuthor: false
+tags: ["News", "ESP-IDF extension", "VSCode", "Visual Studio Code"]
+authors:
+  - "brian-ignacio"
+---
+
+## Introduction
+
+[ESP-IDF VS Code Extension](https://github.com/espressif/vscode-esp-idf-extension) v1.8.0 has been released! It introduces new features and many improvements. If you face any issue or improvement ideas please share them as a [Github issue](https://github.com/espressif/vscode-esp-idf-extension/issues).
+
+This article covers the following key features:
+
+<!-- no toc -->
+- [Eclipse CDT GDB Debug Adapter](#eclipse-cdt-gdb-debug-adapter)
+- [ESP-IDF version switcher](#esp-idf-version-switcher)
+- [Code Navigation without full project build](#code-navigation-without-full-project-build)
+- [ESP-IDF Hints viewer](#esp-idf-hints-viewer)
+
+For other updates please review [v1.8.0 Release notes](https://github.com/espressif/vscode-esp-idf-extension/releases/tag/v1.8.0). Now let's look into these features one by one.
+
+## Eclipse CDT GDB Debug Adapter
+
+First, we will briefly talk about the implementation of debug adapters in VS Code and then will move on to the actual Eclipse CDT Debug Adapter.
+
+### What is a debug adapter in VS Code?
+
+VS Code implements a generic (language-agnostic) _Debugger UI_ that is based on an abstract protocol. That protocol was specifically introduced to communicate with debugger backends. As debuggers typically use their own long-established protocols, some "adapter" is needed to allow debugger UI to communicate with debugger backends.
```

Comment:
> Is it an actual term from VS Code docs? Suggest giving a link. If it is called differently in the VS Code context, suggest making the naming consistent.

This is from my [previous comment](https://github.com/espressif/developer-portal/pull/62#discussion_r1685917878).

Reply:
I think these sentences are taken as they are from vscode documentation regarding debug adapter.

Reply:
Then maybe you can link to that document?

Reply:
done